src/components/RengoAdminModal/RengoAdminModal.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2012-2020  Online-Go.com
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import * as React from "react";
+import {_} from "translate";
+import {post} from "requests";
+import {openModal, Modal} from "Modal";
+import {timeControlDescription, usedForCheating} from "TimeControl";
+import {Player} from "Player";
+import {errorAlerter, yesno} from "misc";
+import swal from 'sweetalert2';
+
+interface Events {
+}
+
+interface RengoAdminModalProperties {
+    challenge: any;
+    onAccept: (challenge) => void;
+    // id?: any,
+    // user?: any,
+    // callback?: ()=>any,
+}
+
+
+export class RengoAdminModal extends Modal<Events, RengoAdminModalProperties, {}> {
```

Comment:
Does this class get used?

Totally possible I'm missing something, but I ctrl+F'd and couldn't find RengoAdminModal or openRengoAdminModal referenced outside of the module itself.

Reply:
Yeah, nah, not used ... should be axed, after we went to all the trouble of working out how to add a new one :D

Reply:
Haha at least ESLint got a fix and no poor souls will have to deal with that in the future >.<